src/repositories_nuxt.tf

```diff
@@ -25,6 +25,9 @@ locals {
       "cargo-shipper" = {
         description = "Cargo Shipper"
       }
+      "services-public" = {
+        description = "Services public promotion website"
+      }
     }
```

Comment:
I wouldn't call it "promotion" if we are not advertising. I think we can call it "Landing page for Services" or "Website about Services".

Reply:
fixed